components/vf-breadcrumbs/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 1.0.1
```

Comment:
1.0.0?

Reply:
I think this commit was meant to in #937, but I don't suppose it really matters.

> Don't ask how the sausage is made.